style/client.css

```diff
@@ -247,7 +247,6 @@ i.subtle:hover {
 	position: relative;
 	padding: 3px 2px 3px 2px;
 	margin: 0 -1px 0 0;
-	top: 1px;
```

Comment:
???

Reply:
I thought the spacing was intentional

Reply:
Sorry, I'm still confused. What is this changing and why? Do you have before/after screenshots?

Reply:
Yeah, 
![image](https://github.com/smogon/pokemon-showdown-client/assets/17451256/8357a061-f83b-4d51-8b14-c61533608b83)
It's the little gap at the top